### PR TITLE
Add location column to elections overview for typist and added missing translations

### LINKED ISSUE #669: Add location column to elections overview for typist
@praseodym noticed that the elections overview for the typist does not show the location for the listed elections. This can be confusing in cases where multiple elections with the same name are active at the same time (ie when a municipality contains different 'waterschappen'. I've added a new column ('gebied') to the column (similar to the elections overview page for coordinators). 

Scope of this ticket:
- add column to the elections overview page for the invoerder
- column should contain the value of the `elections.location` field in the database

### PR DESCRIPTION
**Pull request by Lionqueen94**
Closes #669 

Adds final missing translations
Fixes missing React imports
Fixed footer and i18n mistakes on login and account setup pages

### CODE REVIEW

**Review by praseodym — DISMISSED** [review]

**Review by jorisleker — CHANGES_REQUESTED** [review]
Not sure whether you touched this, but looked into the login form because the file was changed), but spacing between the submit button bar and the form is missing.

Column added to the overview page (scope of the ticket) looks good.
Scherm­afbeelding 2024-12-09 om 15 33 26

**Comment by Lionqueen94** [comment]
> Not sure whether you touched this, but looked into the login form because the file was changed), but spacing between the submit button bar and the form is missing.
> 
> Column added to the overview page (scope of the ticket) looks good. Scherm­afbeelding 2024-12-09 om 15 33 26

Not sure when it was broken, but it's fixed now in 32750aa7624cdd37c3f3e05f87281848aca3c479

**Review by jorisleker — DISMISSED** [review]

**Review by oliver3 — APPROVED** [review]
Many improvements!

File: frontend/app/module/election/page/OverviewPage.tsx [hunk 1]
```diff
@@ -61,15 +63,15 @@ export function OverviewPage() {
         <article>
           <Table id="overview">
             <Table.Header>
-              <Table.Column>{t("election.election")}</Table.Column>
-              {isAdministrator && <Table.Column>{t("role")}</Table.Column>}
+              <Table.Column>{t("election.title.singular")}</Table.Column>
+              <Table.Column>{t(`election.${!isAdministrator ? "location" : "level_polling_station"}`)}</Table.Column>
```
Comment (Lionqueen94) [inline frontend/app/module/election/page/OverviewPage.tsx:67]:
```suggestion
              <Table.Column>{!isAdministrator ? t("election.location") : t("election.level_polling_station")}</Table.Column>
```

File: frontend/app/module/election/page/OverviewPage.tsx [hunk 2]
```diff
@@ -43,7 +45,7 @@ export function OverviewPage() {
       </NavBar>
       <header>
         <section>
-          <h1>{isAdministrator ? t("manage_elections") : t("election.elections")}</h1>
+          <h1>{t(`election.${isAdministrator ? "manage" : "title.plural"}`)}</h1>
```
[the remark this answers is not in the document]
Comment (Lionqueen94) [reply frontend/app/module/election/page/OverviewPage.tsx]: Fair point, I changed both in fe95eecee102cdcbed7582e23a6f3ffa9e669fd4

**Review by praseodym — APPROVED** [review]
lgtm! Next time please make separate pull requests for unrelated changes.

**Pull request merged**